package.json

```diff
@@ -27,13 +27,13 @@
   },
   "devDependencies": {
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
-    "@microsoft/api-extractor": "^7.40.5",
-    "@octokit/core": "^5.1.0",
+    "@microsoft/api-extractor": "^7.43.1",
+    "@octokit/core": "^6.1.2",
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-typescript": "^11.1.6",
     "@types/chai": "^4.3.11",
     "@types/mocha": "^10.0.6",
-    "@types/node": "^20.11.19",
+    "@types/node": "^20.12.7",
     "@types/sinon": "^17.0.3",
     "@types/sinon-chai": "^3.2.12",
     "@typescript-eslint/eslint-plugin": "^7.0.2",
```

Comment:
is there a reason we didn't update the following dependencies?

```typesscript
@typescript-eslint/eslint-plugin     ^7.0.2  →    ^7.7.0
@typescript-eslint/parser            ^7.0.2  →    ^7.7.0
```

Reply:
Good catch.  I updated a few others in addition to these.